source/beamlines.md

```diff
@@ -35,6 +35,7 @@ element in the `line` and with this the length of the BeamLine can be calculated
 The optional `zero_point` component is used to position sublines.
 The value of `zero_point` is the name of a `line item` that marks the reference point. 
 To make things unambiguous, the reference `item` must have zero length.
+[**comment: should the previous sentence be removed?**]
```

Comment:
I think the sentence in line 37 is necessary.  The question is whether the word `item` should be replaced by a different word.  In any case, like 38 should be removed.

Reply:
How about "element or subline"?

Reply:
Suggested changes:  Modify `zero_point` to `reference_point`, and replace all appearances of `line item` and `item` above by the more explicit phrase "zero-length element".

Reply:
@jlvay Are you going to respond to this?